src/romaine/exc.py

```diff
@@ -0,0 +1,7 @@
+# Add Skiptest to exc namespace for more obvious usage
```

Comment:
Can we call this 'exceptions' rather than exc? Parser is using that approach already and I think it's nicer to be excpliit.